version

```diff
@@ -0,0 +1 @@
+3.3.0
```

Comment:
Ahem, liar liar pants on :fire: @bastelfreak. That's still supposed to be "0.3.3".

Reply:
ups, fixed it.